woocommerce/wc-admin
# Add monthly pricing toggle to product types

Fixes #4741

Adds a monthly pricing toggle next to product types as well as popovers for product type descriptions.

### Screenshots
<img width="734" alt="Screen Shot 2020-08-19 at 5 16 55 PM" src="https://user-images.githubusercontent.com/10561050/90690826-06227e00-e27b-11ea-9582-dd99bd510018.png">


### Detailed test instructions:

1. Check that the popovers correctly display the product description and learn more link when clicked.
1. Check that the learn more link still fires the appropriate event.
1. Check that toggling the pricing between monthly and annual shows the correct prices and pill text.
1. Check that style matches expected.

/cc @jameskoster I mostly kept styling consistent with other checkbox groups, but made some stabs at the colors and font-sizes.  Feel free to jump and in and add things or let me know what you'd like to see changed.
